components/tabs/tab-mixin.js

```diff
@@ -0,0 +1,219 @@
+import '../colors/colors.js';
+import { css, html, unsafeCSS } from 'lit';
+import { classMap } from 'lit/directives/class-map.js';
+import { getFocusPseudoClass } from '../../helpers/focus.js';
+import { SkeletonMixin } from '../skeleton/skeleton-mixin.js';
+
+const keyCodes = {
+	ENTER: 13,
+	SPACE: 32
+};
+
+export const TabMixin = superclass => class extends SkeletonMixin(superclass) {
+	static get properties() {
+		return {
+			ariaSelected: { type: String, reflect: true, attribute: 'aria-selected' },
```

Comment:
Is the `ariaSelected` reactive property needed?

We need to set `aria-selected` for a11y (which we're doing below), but we're controlling state using `selected` so this currently seems unnecessary.

Reply:
I was doing this on the off chance that we wanted to more easily access it from external - but I can remove it from the `properties()` hook as I was on the fence about this approach anyway - makes other code more simple too